src/components/WalletDropdown/MiniPortfolio/PortfolioLogo.tsx

```diff
@@ -57,34 +57,23 @@ const ENSAvatarImg = styled.img`
   width: 40px;
 `
 
-const StyledChainLogo = styled.img<{ isSquare: boolean }>`
-  height: ${({ isSquare }) => (isSquare ? '16px' : '14px')};
-  width: ${({ isSquare }) => (isSquare ? '16px' : '14px')};
-  margin-top: ${({ isSquare }) => (isSquare ? '0px' : '1px')};
-  margin-left: ${({ isSquare }) => (isSquare ? '0px' : '1px')};
-  position: absolute;
-  top: 68%;
-  left: 68%;
+const StyledChainLogo = styled.img<{ $padded: boolean }>`
+  height: ${({ $padded }) => ($padded ? '14px' : '16px')};
+  width: ${({ $padded }) => ($padded ? '14px' : '16px')};
```

Comment:
im confused by the convention here - why does `padded` just translate to there being smaller height and width?

Reply:
it does because it's centered relative to the parent, so making it smaller means there is "padding". anyway, i changed this to hopefully be clearer.